pages/about/groups/task-forces/conformance-testing.md

```diff
@@ -32,86 +30,63 @@ Some information on this page is also shown on, and may be more current in, the
+* Give your input promptly, when it’s needed.
 
-## Publications
+To become a participant in the ACT Task Force, you must first be a member of the AG Working Group. See [Instructions for joining the Accessibility Guidelines Working Group](https://www.w3.org/groups/wg/ag/instructions/).
 
-**Accessibility Conformance Testing (ACT) Rules Format 1.0**
+Once you are a member of the AG Working Group, [email the ACT Task Force chairs](https://www.w3.org/groups/tf/wcag-act/participants/#chairs) to let them know you’d like to join the task force. 
 
-- W3C Standard: [https://www.w3.org/TR/act-rules-format/](https://www.w3.org/TR/act-rules-format/)
-- Errata: [https://www.w3.org/WAI/GL/task-forces/conformance-testing/errata](https://www.w3.org/WAI/GL/task-forces/conformance-testing/errata)
+**Note:** As a participant in one of the AG Working Group’s task forces, you can choose to focus your time exclusively on the task force deliverables and you do not have to contribute to the working group’s other deliverables as well.
 
-**Accessibility Conformance Testing (ACT) Rules Format 1.1**
+## Task force members
 
-- First Public Working Draft: [https://www.w3.org/TR/act-rules-format-1.1/](https://www.w3.org/TR/act-rules-format-1.1/)
-- Editor draft: [https://w3c.github.io/wcag-act/act-rules-format.html](https://w3c.github.io/wcag-act/act-rules-format.html)
+* [Chairs](https://www.w3.org/groups/tf/wcag-act/participants/#chairs)
+* [Current participants](https://www.w3.org/groups/tf/wcag-act/participants/#participants)
 
-**Accessibility Conformance Testing (ACT) Rules Repository**
+## Communications
 
-- Published rules: [https://www.w3.org/WAI/standards-guidelines/act/#act-rules-repository](https://www.w3.org/WAI/standards-guidelines/act/#act-rules-repository)
-- GitHub repository: [https://github.com/w3c/wcag-act-rules/](https://github.com/w3c/wcag-act-rules/)
+* [Minutes from previous meetings](https://www.w3.org/WAI/GL/task-forces/conformance-testing/minutes)
+* [Mailing list archive](http://lists.w3.org/Archives/Public/public-wcag-act/)
 
-## How to Comment, Contribute, and Participate
+## Publications and copyright
 
-To join the ACT TF, individuals must be participants of the AG WG. Participants are expected to [actively contribute](/about/groups/task-forces/conformance-testing/work-statement/#participation) to the work of the Task Force. If you are interested in participating in the ACT TF, please send e-mail to: [Daniel Montalvo](mailto:dmontalvo@w3.org?subject=ACT%20TF%20Enquiry) and include a little bit about what you're interested in and how you think that you may be able to contribute to the Task Force. Then follow the [AG Working Group participation](/about/groups/agwg/participate/) procedures to join the Working Group, and once you have joined ask [Daniel Montalvo](mailto:dmontalvo@w3.org) to add you to the task force.
+The ACT Task Force has published the following resources under the name of the AG Working Group:
```

Comment:
This strictly is not correct. AGWG are the publishing group. Might be better to remove the 'and copyright' from the heading - IPR is managed through membership to the WG not the TF. Text might also be better as:

> The ACT Task Force develops and maintains the following resources:

Reply:
I like that wording. Would it be okay to make it: 'The ACT Task Force develops and maintains the following resources published by the AG Working Group.'?
The heading 'Publications' gives the impression that the task force has published the resources, so I'm trying to clarify that task force work is published under a working group name and not the task force name (the relationship of task force work vs working group work is an area of confusion raised numerous times in the user interviews).
This wording would also be helpful to clarify the rare instances when a task force develops and maintains resources for different working groups.